_data/oss_workshop/participants/agarwalanshul.yml

```diff
@@ -0,0 +1,4 @@
+name: Anshul Agarwal
+emoji: 'innocent'
+description: I am a full stack web developer using Microsoft Products.Haven't been able to work on front end and backend for quite some time. 
+Currently working as Service developer using WCF.I am looking to break out of my comfort zone and learn some new cool stuff.
```

Comment:
Need to use a special character to break lines in YAML:

http://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines

but anyway, you're still welcome to come!